src/components/Toast/useToast.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { clickAndWait } from "@homebound/rtl-utils";
+import { useState } from "react";
+import { click, render } from "src/utils/rtl";
+import { Button } from "../Button";
+import { useToast } from "./useToast";
+
+describe("useToast", () => {
+  it("can trigger a toast component", async () => {
+    // Given an app that can trigger a toast
+    const r = await render(<TestToastComponent />);
+    // When triggering the toast
+    click(r.toastButton());
+    // Then expect it to show
+    expect(r.toast()).toBeInTheDocument();
+  });
+
+  it("can close a toast notice", async () => {
+    // Given an app that can trigger a toast
+    const r = await render(<TestToastComponent />);
+    // When triggering the toast and clicking on the close button
+    click(r.toastButton());
+    await clickAndWait(r.toast_close());
```

Comment:
**question:** Does this need to be a `clickAndWait`, or could it have just been a `click`?

Reply:
no, it doesn't need to be a `click`.  I had this as a `clickAndWait` before when I had `onClose` as a callback. I will remove!